docker/odklite/Makefile

```diff
@@ -1,20 +1,30 @@
 
 # Building docker image
-VERSION = "v1.2.27" 
+VERSION = "v1.2.27"
```

Comment:
the VERSION variable should probably be delted and taken from the main Makefile as an ARG?

Reply:
When the `docker/odklite/Makefile` is called from the top-level Makefile, then the top-level Makefile provides the version by overriding that variable (in other words, the `VERSION` variable in the top-level Makefile has precedence over the same variable in the subfolder Makefile).

The rationale for keeping the variable defined in the subfolder Makefile is that it allows to use that Makefile in a "standalone" way, independently of the top-level Makefile. If someone, for any reason, is only interested in building _odklite_, they can do so by going into the `docker/odklite` folder and calling `make` there.

Of course, the version number in the `odklite` Makefile should be kept "in sync" with the version number in the top-level Makefile (otherwise `odklite` would have a different version number depending on whether it has been built from its own Makefile or from the top-level Makefile), which is currently not the case. But we can update that at anytime.

Reply:
Perfect thank you, this makes sense.